src/pubsub/mod_pubsub.erl

```diff
@@ -881,9 +881,9 @@ remove_user_per_plugin_safe(LUser, LServer, Plugin) ->
         plugin_call(Plugin, remove_user, [LUser, LServer])
     catch
         Class:Reason:StackTrace ->
-            ?WARNING_MSG("event=cannot_delete_pubsub_user,"
-                         "luser=~s,lserver=~s,class=~p,reason=~p,stacktrace=~p",
-                         [LUser, LServer, Class, Reason, StackTrace])
+            ?LOG_WARNING(#{what => pubsub_cannot_delete_user, luser => LUser,
```

Comment:
Just `user => LUser, server => LServer`

Reply:
Also, `pubsub_cannot_delete_user` could be converted to `pubsub_delete_user_failed` to be more consistent with the rest of codebase.